src/config/src/meta/stream.rs

```diff
@@ -778,26 +778,24 @@ impl StreamPartition {
     }
 
     pub fn get_partition_value(&self, value: &str) -> String {
-        match &self.types {
+        let val = match &self.types {
             StreamPartitionType::Value => value.to_string(),
```

Comment:
I think we should also do urlencoding on value partition as well.

Reply:
Sorry, this is already handled.

Reply:
Yes